src/rpcserver.cpp

```diff
@@ -282,6 +282,9 @@ static const CRPCCommand vRPCCommands[] =
     { "network",            "getpeerinfo",            &getpeerinfo,            true  },
     { "network",            "ping",                   &ping,                   true  },
 
+    /* ZMQ notification */
+    { "zmq",                "getzmqurl",              &getzmqurl,              true  },
```

Comment:
I didn't say it has to be removed. I just don't see a usecase for this. Maybe @jgarzik can roll this up?